### PR TITLE
WAITP-1367 DataTrak Server

### PR DESCRIPTION
**Pull request by EMcQ-BES**
### Changes:

- Add a new orchestration server package for `datatrak-server`
  - Contains base `login/logout` routes and `getUser`
  - Forwards the rest to central server
  - I've added .env files to lastpass (based on `tupaia-web-server`, most of the fields aren't actually being used here yet)
- Collapse the `mkdir` commands in the `Dockerfile`
  - Dockerfiles can contain a max of `127` layers (commands to layers are vaguely one to one)
  - Reducing the number of commands fixes the build

### CODE REVIEW

File: packages/datatrak-server/.env.example [hunk 1]
```diff
@@ -0,0 +1,12 @@
+DB_NAME=
```
Comment (tcaiger) [inline packages/datatrak-server/.env.example:1]: Just double checking that you got around to adding in the env files in last pass? :)

File: packages/datatrak-server/src/models/DataTrakSession.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { SessionType, SessionModel } from '@tupaia/server-boilerplate';
+
+export class DataTrakSessionType extends SessionType {
+  public static databaseType = 'datatrak_session';
+}
+
+export class DataTrakSessionModel extends SessionModel {
```
Comment (tcaiger) [inline packages/datatrak-server/src/models/DataTrakSession.ts:12]: I guess we will need to add the hardcoded public permissions that tupaia users have in to here somewhere as well. I'll make a note on the ticket for that one.

File: packages/datatrak-server/.env.example (hunk 1 above)
Comment (EMcQ-BES) [reply packages/datatrak-server/.env.example:1]: image

Comment (tcaiger) [reply packages/datatrak-server/.env.example:1]: 🤦

**Review by tcaiger — APPROVED** [review]

**Pull request merged**